RoddyCore/src/de/dkfz/roddy/client/RoddyStartupModes.groovy

```diff
@@ -111,8 +111,8 @@ enum RoddyStartupModes {
         System.out.println(String.format("  %s %s " + Constants.ENV_LINESEPARATOR + "\t%s" + Constants.ENV_LINESEPARATOR, option.toString(), parameters, formattedDescription.toString()))
     }
 
-    static void printCommands(List<Object[]> commands) {
-        commands.each { Object[] it ->
+    static void printCommands(List<List<Object>> commands) {
+        commands.each { List<Object> it ->
             printCommand(it[0] as RoddyStartupModes, it[1] as String, it[2..-1] as String[])
```

Comment:
Unfortunately, I could not come up with a type safe version without adding something like
class CommandPrintArgument {
RoddyStartupModes mode;
String options;
String[] additionalMessages;
}

Would be a lot more code. Suggestions? We could maybe convert it to List<String>. That is still not type safe but we convert every entry to string anyways.